capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './nameable'
+class CapitalizeDecorator < Nameable

```

Comment:
- [ ] According to the project requirements, the CapitalizeDecorator  should Inherit from `BaseDecorator` instead of the Nameable class
* Check `CapitalizeDecorator` class, it should look similar to:

```ruby
# Inherits from Decorator
class CapitalizeDecorator < BaseDecorator
    require './base_decorator'

    # Capitalize the first letter of the given name.
    def correct_name
      @nameable.correct_name.capitalize
    end
end
```